### DISCUSSION TITLE
Verification not working

### DISCUSSION THREAD

**Discussion by nihilisticzebra** [community/community#117450] [GitHub Education]
I am already part of the program I'm trying to reverify my status but none of my screenshots works ( using my schedule that shows I'm enrolled and current status including semester dates). I can only take pictures and can't upload documents. I've tried everything but I keep getting this error 
![image](https://github.com/community/community/assets/65736687/1b152d0f-ebe3-4c67-9adb-5188a5e0efe8)

**Comment by nihilisticzebra** [comment]
> I am already part of the program I'm trying to reverify my status but none of my screenshots works ( using my schedule that shows I'm enrolled and current status including semester dates). I can only take pictures and can't upload documents. I've tried everything but I keep getting this error 
> ![image](https://github.com/community/community/assets/65736687/1b152d0f-ebe3-4c67-9adb-5188a5e0efe8)

Looks like it's processing now after sending it like 12 times, in 2022 I had to get it manually approved but now this feature was removed hopefully it works this time.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education. 🎉
